pkg/cmd/install.go

```diff
@@ -414,6 +415,24 @@ func (o *installCmdOptions) setupOperator(
 			platformName)
 	}
 
+	// Choose an architecture specific operator/base image when not already given explicitly
+	//
+	arch, err := kubernetes.NodeArchLookup(o.Context, c)
```

Comment:
I don't think the CLI should inspect the node. You're assuming it has privileges that it should not have.

Reply:
I assume that 

```
kamel install 
```

works, when you first come to this project. Folks (like me) try this out on their MacBook and we may never hear why they turned around after it didn't. This functionality can of course improve over time, For now however, the goal is to make it work for most.

Reply:
Adding any code it means maintenance burden in the long run to the people maintaining the project. We must be careful to add only whatever is meant to be supported. We have a procedure to install the ARM based architecture [1] and beside that the final solution has been already identified and provided in a comment above as well.

[1] https://camel.apache.org/camel-k/2.2.x/installation/advanced/multi-architecture.html